src/Commands/DibsCommand.php

```diff
@@ -114,6 +114,8 @@ public function siteDibsAny($site, $message, $filter = '^((?!^live$).)*$') {
    * @param string $filter An optional regex pattern used to filter the pool of
    *   environments for which you wish to view the report.
    *
+   * @param integer $duration An optional time threshold (seconds) for duration that an environment has been dibs'd.
```

Comment:
Since this is optional, it'd make sense to use a flag instead of an argument: `terminus site:dibs:report [site] --duration=[n seconds]`

And with that, maybe `--older-than=[n seconds]` to be explicit about what the duration represents.

Here's an example in Terminus core: https://github.com/pantheon-systems/terminus/blob/master/src/Commands/Multidev/DeleteCommand.php#L27